### PR TITLE
Nighthawk/mitigation review update manage findings

### PR DESCRIPTION
**Pull request by hannahtuttle**

### CODE REVIEW

**Comment by hannahtuttle** [comment]
@captainmangoC4 here are the changes for the manage findings, i also had to make some small updates to the submit findings because i was missing some data being passed down to the manage findings related to mitigationOf.

**Comment by hannahtuttle** [comment]
@captainmangoC4 I made those updated for changing a mitigation from a new risk to a confirmed mitigation.

File: src/components/ContestTile.js [hunk 1]
```diff
@@ -84,20 +84,20 @@ const ContestTile = ({ contest, updateContestStatus, user }) => {
               View Repo
             </a>
           )}
-          {(t.contestStatus === "active" || status === "Active Contest") &&
+          {/* {(t.contestStatus === "active" || status === "Active Contest") &&
```
Comment (captainmangoC4) [inline src/components/ContestTile.js:87]: You can uncomment these lines.

File: src/components/reporter/SubmitFindings.tsx [hunk 2]
```diff
@@ -333,12 +331,13 @@ const SubmitFindings = ({
...
     const markdownBody = `# Lines of code\n\n${linksToCodeString}\n\n\n# Vulnerability details\n\n${state.details}`;
-
     let risk = state.risk;
     let title = getTitle(state.title, state.risk);
     let body = markdownBody;
     let labels = [config.labelAll, state.risk];
-    let mitigationOf: string | undefined = undefined;
+
+    //test here to make sure mitigigation of on a regular submit and edit are working correctly
+    let mitigationOf: string | undefined = state.mitigationOf ? state.mitigationOf : undefined;
```
Comment (captainmangoC4) [inline src/components/reporter/SubmitFindings.tsx:340]: Can this be reverted now?

**Comment by hannahtuttle** [comment]
@captainmangoC4 this is ready for you again!

**Review by captainmangoC4 — APPROVED** [review]

**Pull request merged**